src/imputation_flags.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+
+from src.flag_and_count_matched_pairs import flag_matched_pair_merge
+
+
+def create_impute_flags(df, target, reference, strata, auxiliary):
```

Comment:
Not sure if we should consider splitting it into 4 functions create r_flags,fir_flags etc and having them in a separate module